### ISSUE TITLE
Add Palettes Window Not Read by Screen Readers on Windows

### ISSUE THREAD

**Issue by NoahTCarver** [musescore/MuseScore#16744]
### Issue type

UI bug

### Bug description

The "Add Palettes" window is not read by screen readers on Windows. I have only been able to focus the buttons contained within once, and have no idea how I did so. The window's contents are apparently always read on macOS and Linux as evidenced by both conversations in the MuseScore discord server and MuseScore Accessibility WhatsApp group.

### Steps to reproduce

1. Launch a Windows screen reader (Narrator or NVDA are easiest to obtain).
2. Launch MS 4 and create/open a score.
3. Use the keyboard to navigate to the "Add Palettes" button. Activate it with space/enter.
4. Screen reader reports some form of text "MuseScore4  window."
5. No further keyboard navigation is possible; escape key is only exit.
6. If screen reader does allow nagivation, I'd love to know how you did it.Version: 4.0.2.230601206
Revision: 48ac167
Version: 4.0.2.230601206
Revision: 48ac167

### MuseScore Version

Version: 4.0.2.230601206 Revision: 48ac167

### Regression

I don't know

### Operating system

Edition: Windows 11 ProVersion	22H2 Installed on	‎10/‎11/‎2022 OS build	22621.1265 Experience	Windows Feature Experience Pack 1000.22638.1000.0

### Additional context

Tested using: NVDA 2022.4 (2022.4.0.27401), JAWS 2023, Narrator

**Comment by zacjansheski** [comment]
Similar issue
https://github.com/musescore/MuseScore/issues/17922

Works as desired in Mac OS with VoiceOver.  Does not work as desired in Windows with Narrator or NVDA

**Comment by jmwmusic** [comment]
Palette More window and Instrument Settings Panel also remain exactly the same as this with nothing reading and no result from attempts to Tab or arrow.  The fact that OCR can be used will not be known by many NVDA users (I didn't ever think of how to try this)
#17922 may therefore need reopening, or should I create a new issue for each, these 3 seem to be closely related.

**Comment by jmwmusic** [comment]
Is this the same issue as the appearance window not reading #16739?  As blind users this and the other 3 similar have been reported as not being able to navigate, but it seems now it is probably a lack of speech feedback but navigation is taking place, all be it with us unaware!

**Comment by MarcSabatella** [comment]
Yes, as noted in the description, other screen readers work fine with this panel.  I think the issue is probably unique to NVDA (and maybe Narrator), and is probably the same root cause as for other panels that show similar symptoms.

**Comment by jmwmusic** [comment]
Further testing has shown that this and other related panels such as the Instrument Properties panel do navigate with keyboard (except for tab not working as all items are in a single group) and also reads with the JAWS screen reader on Windows, but still no speech with NVDA.  Note most of us do not have access to JAWS and it generally doesn't work well with MuseScore Studio so this is just a testing point.

There is also a sub dialogue with the same issue that opens here if you press Create Custom Palette.

**Comment by jmwmusic** [comment]
The following seem to behave similarly:
#25404 instrument Properties
#16092 Palette More
#16739 Appearance window (mentioned in comments of this ticket)

**Comment by jmwmusic** [comment]
This seems to be now reading in 4.7 beta with NVDA.  The sub dialogue for the create custom Palette though does not.

**Comment by jmwmusic** [comment]
I can confirm this area is now reading with NVDA in 4.7.
